core/build.gradle

```diff
@@ -36,11 +36,11 @@ archivesBaseName = 'applicationinsights-core'
 
 dependencies {
     provided (project(':agent')) { transitive = false }
-    compile group: 'eu.infomas', name: 'annotation-detector', version: '3.0.4'
-    compile group: 'commons-io', name: 'commons-io', version: '2.4'
-    compile group: 'org.apache.commons', name: 'commons-lang3', version: '3.1'
-    compile group: 'org.apache.httpcomponents', name: 'httpclient', version: '4.3.5'
-    compile group: 'com.google.guava', name: 'guava', version: '12.0.1'
+    compile ([group: 'eu.infomas', name: 'annotation-detector', version: '3.0.4'])
+    compile ([group: 'commons-io', name: 'commons-io', version: '2.4' ])
+    compile ([group: 'org.apache.commons', name: 'commons-lang3', version: '3.1']) 
+    compile ([group: 'org.apache.httpcomponents', name: 'httpclient', version: '4.5.3'])
```

Comment:
I remember there were some version specific things in our code around apache.http - did you check that code and telemetry transmission are fine with that change?